File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 4/SOAL 1/PRAK401-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main() {
    int awal = 1, kelipatan;
    char character;
    scanf("%d %c", &kelipatan, &character);
    while (awal != 51) {
        if (awal % kelipatan != 0) {
            printf("%d ", awal);
        } else {
            printf("%c ", character);
        } awal++;
    } return 0;
}

File: MODUL 4/SOAL 1/PRAK401-2410817210002-Sebastian_Abe_Santoso.py
a = []
while len(a) != 2:
    b = input()
    a.extend(b.split())
kelipatan, character = a[:2]
kelipatan = int(kelipatan)
awal = 1
while awal != 51:
    if awal % kelipatan != 0:
        print(awal, end = ' ')
        awal += 1
    else:
        print(character, end = ' ')
        awal += 1